service_container/service_subscribers_locators.rst

```diff
@@ -122,6 +122,24 @@ It is also a callable and a countable::
 
     return $handler->handle($command);
 
+Because the service locator implements the
+:class:`Symfony\\Contracts\\Service\\ServiceCollectionInterface`, it is also
+a ``\Traversable``. This allows to iterate through all registered services to
+apply some operations on them::
+
+    foreach ($this->locator as $serviceId => $service) {
+        // access to the service directly
+        $service->someMethod();
+
+        // or via the `get()` method
+        $this->locator->get($serviceId)->someMethod();
```

Comment:
Let's move this line out of the foreach as a separate example. `$service` is initialized at this point.